<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AsSettingsView</class>
 <widget class="QWidget" name="AsSettingsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>448</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Courier New</family>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 3px;
    margin-top: 0.5em;
    font: bold &quot;Times New Roman&quot;;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="generalBox">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>General</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Additional Launch Parameters</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="parametersLineEdit"/>
      </item>
      <item>
       <widget class="QCheckBox" name="versionCheckBox">
        <property name="text">
         <string>Enable version check at startup</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="pathsBox">
     <property name="title">
      <string>Paths</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="PathSetting" name="steamPathSetting" native="true"/>
      </item>
      <item>
       <widget class="PathSetting" name="arma3PathSetting" native="true"/>
      </item>
      <item>
       <widget class="PathSetting" name="modsPathSetting" native="true"/>
      </item>
      <item>
       <widget class="PathSetting" name="teamspeakPathSetting" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="networkBox">
     <property name="title">
      <string>Network Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="OptionalSetting" name="uploadSetting" native="true"/>
      </item>
      <item>
       <widget class="OptionalSetting" name="downloadSetting" native="true"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="portLineEdit"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="deltaPatchingCheckbox">
        <property name="text">
         <string>Enable delta patching</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="troubleshootingBox">
     <property name="title">
      <string>Troubleshooting</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="reportButton">
        <property name="text">
         <string>Report Bug..</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OptionalSetting</class>
   <extends>QWidget</extends>
   <header>src/view/optionalsetting.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PathSetting</class>
   <extends>QWidget</extends>
   <header>src/view/pathsetting.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
